Document/0x06g-Testing-Network-Communication.md

```diff
@@ -314,6 +314,12 @@ You can look into the [pinning.ts](https://github.com/sensepost/objection/blob/m
 
 See also [Objection's documentation on Disabling SSL Pinning for iOS](https://github.com/sensepost/objection#ssl-pinning-bypass-running-for-an-ios-application "Disable SSL Pinning in iOS" ) for further information.
 
+However, technologies and systems change over time, and this bypass technique might not work eventually. Hence, it is part of the tester work to do some research, as not every tool keeps up OS versions quickly enough.
```

Comment:
```suggestion
However, technologies and systems change over time, and this bypass technique might not work eventually. Hence, it's part of the tester work to do some research, since not every tool is able to keep up with OS versions quickly enough.
```